src/Authentication/Actions/Email2FA.php

```diff
@@ -71,14 +71,13 @@ public function handle(IncomingRequest $request)
 
         // Send the user an email with the code
         helper('email');
-        $return = emailer()->setFrom(setting('Email.fromEmail'), setting('Email.fromName') ?? '')
-            ->setTo($user->email)
-            ->setSubject(lang('Auth.email2FASubject'))
-            ->setMessage(view(setting('Auth.views')['action_email_2fa_email'], ['code' => $identity->secret]))
-            ->send();
-
-        if ($return === false) {
-            throw new RuntimeException('Cannot send email for user: ' . $user->email);
+        $email = emailer()->setFrom(setting('Email.fromEmail'), setting('Email.fromName') ?? '');
+        $email->setTo($user->email);
+        $email->setSubject(lang('Auth.email2FASubject'));
+        $email->setMessage(view(setting('Auth.views')['action_email_2fa_email'], ['code' => $identity->secret]));
+
+        if ($email->send(false) === false) {
+            throw new RuntimeException('Cannot send email for user: ' . $user->email . ' ' . $email->printDebugger(['headers']));
```

Comment:
How about this?

```suggestion
            throw new RuntimeException('Cannot send email for user: ' . $user->email . "\" . $email->printDebugger(['headers']));
```

Reply:
This is good. 
Thank you.

Reply:
Just tried this, but it is returning syntax error. Is this what you were trying to explain:
```
throw new RuntimeException('Cannot send email for user: ' . $user->email . '"\"' . $email->printDebugger(['headers']));
```

Reply:
Oh, sorry, I missed `n`. I mean `"\n"`.

Reply:
Thank you.